axum/src/extract/multipart.rs

```diff
@@ -142,6 +142,38 @@ impl<'a> Field<'a> {
     pub async fn text(self) -> Result<String, MultipartError> {
         self.inner.text().await.map_err(MultipartError::from_multer)
     }
+
+    /// Stream a chunk of the field data.
+    ///
+    /// When the field data has been exhausted, this will return [`None`].
+    ///
+    /// This does the exact same thing as the `Stream` impl and is for convenience
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use axum::{
+    ///    extract::Multipart,
+    ///    routing::post,
+    ///    Router,
+    /// };
+    ///
+    /// async fn upload(mut multipart: Multipart) {
+    ///     while let Some(mut field) = multipart.next_field().await.unwrap() {
+    ///         while let Some(chunk) = field.chunk().await.unwrap() {
+    ///             println!("Chunk: {:?}", chunk);
```

Comment:
I think the data is likely to be binary. So perhaps just printing the length is better 🤔 

```suggestion
    ///             println!("received {} bytes", chunk.len());
```

Reply:
changed